tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_2017_gfdlmp.yaml

```diff
@@ -1,27 +0,0 @@
-metadata:
-  description: |-
-    This test is to ensure that the workflow running in community mode
-    completes successfully on the RRFS_CONUS_25km grid using the GFS_2017_gfdlmp
-    physics suite with ICs and LBCs derived from the FV3GFS.
-    This test additionally includes the cycling ensemble capability, with
```

Comment:
The cycling ensemble capability needs to be added to another test with the loss of this one. I am not sure what the best candidate might be; there's also the option to just remove it temporarily and add it back with a future PR (my upcoming testing PR adds a few new tests)

Also, The test `wflow_features/config.community_ensemble_2mems.yaml` is a symlink to this one; you should replace that symlink to point to the test above.

Reply:
I have added the test back, but am now having it use `FV3_RAP` CCPP physics suite.  I have also changed the symlink for `wflow_features/config.comunity_ensemble_2mems.yaml` to point to the new test.  I also noted that `wflow_feature/config.specify_RESTART_INTERVAL.yaml` was pointing at a test configuration that no longer exists, so I updated the symlink for this configuration too.